### PR TITLE
readme added

### PR DESCRIPTION
**Pull request by jiwonkim19**
Added instructions to readme.md file. Try to install and let me know if you have any problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,79 @@
...
+  $ python3 -m venv env
+
+* Activate a virtual environment: 
+  $ source env/bin/activate
+
+* To leave a virtual environment:
+  $ deactivate
+
+* Install dependencies: 
+  $ python3 -m pip install < backend-dependency>
```
Comment (eternal44) [inline README.md:32]: I found this command that will install everything in the `requirements.txt` file with one command:
```
python3 -m pip install -r requirements.txt
```

File: README.md [hunk 2]
```diff
@@ -1,42 +1,79 @@
...
+  $ npm install <frontend-dependency>
```
Comment (eternal44) [inline README.md:33]: Same - you can run `npm install` and it'll install everything in your `package.json` file.

File: README.md [hunk 3]
```diff
@@ -1,42 +1,79 @@
...
+          - Axios
+          - React-router-dom
+          - nodemon
+
+
+* Run the following to run build after each file save in src: 
+  $ nodemon -w src -x "npm run build" 
+
+
+* Create a database in Postgres
```
Comment (eternal44) [inline README.md:59]: What should the database be called?

Also, adding the command to create the db would be nice. Something like:
```
createdb joytech_development
```

or something.

File: README.md [hunk 4]
```diff
@@ -1,42 +1,79 @@
...
+Built with:
+  * Django
+  * React
+
+# Installation:
+
+* To see which version of Python3 you have installed:
+  $ python3 --version
+
+* Install Python 3.9.10:
```
Comment (eternal44) [inline README.md:15]: I couldn't install 3.9.10 so I'm going to try with 3.8.10... hopefully it just works :P

File: README.md (hunk 1 above)
Comment (jiwonkim19) [reply README.md:32]: ok. updated

File: README.md (hunk 2 above)
Comment (jiwonkim19) [reply README.md:33]: updated

File: README.md (hunk 3 above)
Comment (jiwonkim19) [reply README.md:59]: Hey! So I gave the dev the freedom to name their own database. As long as it correlates with the .env file, they should be fine.

Comment (eternal44) [reply README.md:59]: Sounds good!

**Review by eternal44 — CHANGES_REQUESTED** [review]
I was able to spin up the frontend and backend servers as well as create a request - looks like I'm good to start!

I'll keep an eye out on my different python version. If it becomes a problem down the line we can tackle it then.

Lastly, can you add the command to create a database? Everything else LGTM!

File: README.md (hunk 4 above)
Comment (jiwonkim19) [reply README.md:15]: Yep. Let me know if you run into any trouble.

**Pull request closed without merging**